packages/sdk/tasks/register-erc20.ts

```diff
@@ -0,0 +1,62 @@
+import '@nomiclabs/hardhat-ethers'
+import {predeploys} from '@wemixkanvas/contracts'
+import {task, types} from 'hardhat/config'
+import 'hardhat-deploy'
+
+const SIMPLE_FACTORY_ABI = [
+  'event KanvasMintableERC20Created(address indexed localToken, address indexed remoteToken, address deployer)',
+  'function createKanvasMintableERC20(address _remoteToken, string memory _name,string memory _symbol) public returns (address) ',
+]
+const SIMPLE_ERC20_ABI = [
+  'function name() view returns (string)',
+  'function symbol() view returns (string)',
+  'function decimals() view returns (uint8)',
+]
```

Comment:
To maintain forward compatability, you should get ABI from contract definition like how `deposit-eth` does.

```typescript
const Artifact__L2ToL1MessagePasser = await getContractDefinition(
  'L2ToL1MessagePasser'
)
```